### ISSUE TITLE
[a11y]: Run accessibility checks on all views and file issues for any findings

### ISSUE THREAD

**Issue by lsr-explore** [usdigitalresponse/usdr-gost#3758]
### Why is this issue important?

This is important to be able to identify any barriers customers using assistive technologies will face.

## Accessibility Checks
* axe devTools scan
* Keyboard only navigation
* Screen reader navigation - check rotor menus

## Views
* Dashboard
  * Submit workbook
* Uploads
  * Edit
  * Upload Period Template
* Agencies
  * Create new agency 
* Subrecipients
* Users
  * Create new user 
* Reporting Periods
  * Create new reporting period 
* Validation

## Notes
Capturing items found during initial checks

### Small effort
* Color contrast issues on buttons with blue backgrounds
* Edit buttons need differentiation - add aria-label that includes name
  * On the rotor menu, there is just a list of buttons all with the same name "Edit"
* Upload template links need differentiation - add aria-label that includes name
  * On the rotor menu, there is just a list of buttons all with the same name "Edit"

### Large effort
* Tables need to be horizontally scrolled when view is zoomed 200x

### Current State

When user visits X page...

### Expected State

When user visits X page...

### Implementation Plan

The following functions need to be added...

**Comment by lsr-explore** [comment]
# Uploads View

## axe-dev-tools

### Critical issue
![Image](https://github.com/user-attachments/assets/28ab049a-d4d7-47ab-a98d-75a6b735684e)

### Serious Issue
![Image](https://github.com/user-attachments/assets/39570a2a-57b2-4e7f-aa10-dff5e26d2708)

![Image](https://github.com/user-attachments/assets/903c1db6-d207-4d02-89a1-f34f097a816f)

![Image](https://github.com/user-attachments/assets/94798f97-dcef-484f-821a-cf382aca21bf)


### Minor Issue
![Image](https://github.com/user-attachments/assets/bd8ccc94-236d-4d36-b220-1e41cb5ba885)

**Comment by lsr-explore** [comment]
# Agencies

## axe-dev-tools

### Serious issue
![Image](https://github.com/user-attachments/assets/9f449dbb-9541-4f9d-a57b-a54f933dd378)

**Comment by lsr-explore** [comment]
# Subrecipients

## axe-dev-tools

### Serious issue
![Image](https://github.com/user-attachments/assets/a1460e8a-ba82-4ecf-95fe-0db543d88324)

![Image](https://github.com/user-attachments/assets/f23207d8-cf8b-4608-8026-9bc55cc1e5c2)

### Minor issue
![Image](https://github.com/user-attachments/assets/e2db4fcb-7ce6-43a3-b6d7-15ee000a5dfb)

**Comment by lsr-explore** [comment]
# Users

## axe-dev-tools

### Serious Issue
![Image](https://github.com/user-attachments/assets/06057d93-1b02-4944-bceb-27178f3686f4)

**Comment by lsr-explore** [comment]
# Reporting Periods

## axe-dev-tools

### Serious Issue
![Image](https://github.com/user-attachments/assets/7fb7c451-df68-47fc-b705-591ecf4b81d9)

![Image](https://github.com/user-attachments/assets/b4ec0339-fb09-4df8-8bc6-80758aebd5a8)

![Image](https://github.com/user-attachments/assets/72bbfdc0-f932-45e4-b4f5-1c7007dacc23)


### Minor Issue
![Image](https://github.com/user-attachments/assets/579de7a8-7cc8-4724-88d0-6e963c84a72d)


----

# Create new Reporting Period

## axe-dev-tools

### Serious issue
![Image](https://github.com/user-attachments/assets/042777ce-1e68-4437-8ee4-bde140a45c08)

**Comment by lsr-explore** [comment]
# Validation

## axe-dev-tools

### Serious issue
![Image](https://github.com/user-attachments/assets/5a369d26-5b8d-4852-b7d3-67de969794d5)

**Comment by lsr-explore** [comment]
# Keyboard navigation

Focus indicator
* All views w/ tables - column sort buttons missing sort indicator
* Creating new views - No way to easily return to table views after edit (need to check create)
  * Add button to cancel
  * After add - return to views

**Comment by lsr-explore** [comment]
Marking this as complete.   Initial review of views completed.  Links to the issues filed posted above.  No code changes are associated with this ticket.
